android/app/src/main/java/org/fossasia/openevent/fragments/AboutFragment.java

```diff
@@ -131,47 +130,43 @@ private void loadEvent(Event event) {
                 DateUtils.formatDateWithDefault(DateUtils.FORMAT_DATE_COMPLETE, event.getStartTime()),
                 DateUtils.formatDateWithDefault(DateUtils.FORMAT_DATE_COMPLETE, event.getEndTime()));
 
-        welcomeMessage.setText(getText(R.string.welcome_message));
-        Views.setHtml(organiser_description, event.getOrganizerDescription(), true);
-        Views.setHtml(event_description, event.getDescription(), true);
-        venue_details.setText(event.getLocationName());
-        event_timing.setText(date);
+        welcomeMessage.setText(event.getName());
```

Comment:
I think its better to maintain the original message with "Welcome to (event name)!" instead of just the event name.